### PR TITLE
feat(FN-2594): remove disabled year and autocomplete fix on find report by year tfm page

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction :pencil2:
This PR removes the disabled year on the find report by year TFM page and removes previous entries being added to the autocomplete

## Resolution :heavy_check_mark:
* Added autocomplete: off to the text input
* Removed disabled from `enableFindReportsByYearDropdown`

## Miscellaneous :heavy_plus_sign:
* Updated integrity for script

### CODE REVIEW

File: e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/find-utilisation-reports-by-bank-and-year.spec.js [hunk 1]
```diff
@@ -85,13 +85,13 @@ context('PDC_RECONCILE users can search for reports by bank and year', () => {
     pages.searchUtilisationReportsResultsPage.noReportsText().should('exist');
   });
 
-  it('should disable the year input until a bank is selected', () => {
+  it('should NOT disable the year input and should NOT have any dropdown options when no bank is selected', () => {
```
Comment (ttbarnes) [inline e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/find-utilisation-reports-by-bank-and-year.spec.js:88]: Split into 2x tests?

File: trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk [hunk 2]
```diff
@@ -87,5 +88,5 @@
 {% block sub_content %}{% endblock %}
 
 {% block page_scripts %}
-  <script src="/assets/js/enableFindReportsByYearDropdown.js" integrity="sha512-QLvf2zIp3WAaUOyTIf4SlZ9JLBWVw1CCwwY8Ozzc3+mTsFv4WClwZ9ZYNdNwf9nSFNd2Y4WNr1GL/Bc6ABo1Fw==" type="text/javascript" crossorigin="anonymous"></script>
+  <script src="/assets/js/enableFindReportsByYearDropdown.js" integrity="EFJiI4d7Q8VRfgYiiboT3GIz93joREbGYasuLLcdGROS7srBMp+Y+b9b5U389AXQhsDbAkuSwnZtEmnRUWrYSg==" type="text/javascript" crossorigin="anonymous"></script>
```
Comment (ttbarnes) [inline trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk:91]: If not already, can you check the other script's integrity hashes? `mojFilterHide` and `ssoRedirectAutoSubmit have been re-generated

**Review by ttbarnes — CHANGES_REQUESTED** [review]

File: trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk (hunk 2 above)
Comment (Zainzzkk) [reply trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk:91]: updated for mojFilterHide.  Cannot find any integrity hashes for ssoRedirectAutoSubmit

Comment (ttbarnes) [reply trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk:91]: @Zainzzkk it's missings the `sha512-` prefix?

Comment (Zainzzkk) [reply trade-finance-manager-ui/templates/utilisation-reports/find-utilisation-reports-by-year.njk]: added back

**Review by ttbarnes — APPROVED** [review]

**Review by ChristianMcc — APPROVED** [review]
@AlexBramhill we have rebuilt an SSO script but haven't regenerated the integrity and need to get this in. Do you mind dealing with if you need to rebuild the integrity for your work separately please? just giving you a heads up!

**Pull request merged**